js_modules/dagit/packages/core/src/nav/useRepositoryLocationReload.tsx

```diff
@@ -120,21 +168,25 @@ export const useRepositoryLocationReload = (location: string) => {
 
         // On success, show the successful toast, hide the dialog (if open), and reset Apollo.
         SharedToaster.show({
-          message: 'Repository location reloaded!',
+          message: `${scope} reloaded!`,
```

Comment:
Consider splitting the enum values from the actual rendered strings? Just in case we need to display something more complicated in certain cases.